daemons/attrd/attrd_corosync.c

```diff
@@ -257,6 +257,9 @@ update_attr_on_host(attribute_t *a, const crm_node_t *peer, const xmlNode *xml,
 
     v = attrd_lookup_or_create_value(a->values, host, xml);
 
+    crm_debug("Hit local sync point for attribute %s", attr);
+    attrd_alert_waitlist_clients(attrd_sync_point_local, attr);
```

Comment:
I'd avoid the term "alert" here since there are formal attribute alerts that use alert agents. "notify" is what we typically use for such things internally, though "ack" might make more sense in this particular case